File: app/src/main/java/com/example/indiwarenative/lesson.java
package com.example.indiwarenative;

import java.sql.Date;
import java.sql.Time;
import java.time.LocalTime;

public class lesson {
    public int pos;
    public String teacher;
    public String subject;
    public String room;
    public LocalTime start;
    public LocalTime end;
    public boolean canceled;

    public lesson(int pos, String teacher, String subject, String room, LocalTime start, LocalTime end, boolean canceled) {
        this.teacher = teacher;
        this.subject = subject;
        this.room = room;
        this.start = start;
        this.end = end;
        this.pos = pos;
        this.canceled = canceled;
    }

    public  lesson () {
        this.pos = 0;
        this.subject = "error";
        this.room = "error";
        this.teacher = "error";
    }
}

File: app/planagerWear/src/main/java/com/example/planager/lesson.kt
package com.example.planager

import kotlinx.serialization.Serializable
import java.time.LocalTime

@Serializable
data class lesson(
    val pos: Int = 0,
    val teacher: String = "error",
    val subject: String = "error",
    var room: String = "error",
    val roomChanged: Boolean = false,
    val start: LocalTime? = null,
    val end: LocalTime? = null,
    val canceled: Boolean = false,
    val ag: Boolean = false
)
